app/scheduler.py

```
@@ -0,0 +1,63 @@
+    def schedule_post(self, post: ScheduledPost) -> None:
+        """Database persistence already stores jobs; method kept for parity."""
+
+    def _run(self) -> None:
+        while not self._stop_event.is_set():
+            self.dispatch_due_posts()
+            self._stop_event.wait(self.poll_seconds)
+
+    def _publish_post(self, post: ScheduledPost) -> None:
+        try:
+            self.twitter_client.post_tweet(post.content)
+        except Exception as exc:  # pragma: no cover - best-effort logging
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)